src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/login/CustomPagingNavigatorSpec.groovy

```diff
@@ -0,0 +1,47 @@
+package pl.tomaszdziurko.jvm_bloggers.view.login
+
+import org.apache.wicket.Component
+import org.apache.wicket.markup.repeater.data.DataView
+import org.apache.wicket.model.Model
+import org.apache.wicket.protocol.http.WebApplication
+import org.apache.wicket.util.tester.WicketTester
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.test.util.ReflectionTestUtils
+import pl.tomaszdziurko.jvm_bloggers.SpringContextAwareSpecification
+import pl.tomaszdziurko.jvm_bloggers.view.login.helpers.TestDataNavigationPage
+
+class CustomPagingNavigatorSpec extends SpringContextAwareSpecification {
```

Comment:
After introducing a new way of Wicket components testing in #182 there is no need to start-up entire Spring context. You can just extend [`MockSpringContextAwareSpecification`](https://github.com/tdziurko/jvm-bloggers/blob/master/src/test/groovy/pl/tomaszdziurko/jvm_bloggers/MockSpringContextAwareSpecification.groovy). See [`LoginPageSpec`](https://github.com/tdziurko/jvm-bloggers/blob/master/src/test/groovy/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPageSpec.groovy) for example usage.